lib/cib/cib_remote.c

```diff
@@ -328,6 +368,7 @@ cib_tls_signon(cib_t *cib, pcmk__remote_t *connection, gboolean event_channel)
 
     struct remote_cb_data_s *rd = pcmk__assert_alloc(1, sizeof(struct remote_cb_data_s));
     rd->cib = cib;
+    rd->start_time = time(NULL);
```

Comment:
The point of creating this was to pass it as data to the cib_fd_callbacks way at the bottom of this function.  However I've gotten pretty lost in the maze of callbacks and timeouts.  Are you saying I can just completely get rid of this data struct?

Reply:
I was confused. This is async as far as dispatch goes, but the connection is sync. There's no connection callback, so it's fine to leave it that way.

I don't think we need to initialize start_time here, though. We do send a registration message, so probably it would be fine, but logically, we're not starting a new message here that must be completed in a certain time.